azure-pipelines.yml

```diff
@@ -0,0 +1,27 @@
+# Starter pipeline
```

Comment:
I would say to delete the comment lines as long as they are not related to our YAML file.